armi/reactor/reactorParameters.py

```diff
@@ -723,42 +770,47 @@ def defineCoreParameters():
 
         pb.defParam(
             "breedingRatio",
-            units="N/A",
+            units=units.UNITLESS,
             description="Breeding ratio of the reactor",
             default=0.0,
         )
 
-        pb.defParam("ConvRatioCore", units="?", description="?")
+        pb.defParam(
+            "ConvRatioCore",
+            units=units.UNITLESS,
+            description="Conversion ratio of the core",
+        )
 
-        pb.defParam("absPerFisCore", units="?", description="?")
+        pb.defParam(
+            "absPerFisCore",
+            units=units.UNITLESS,
+            description="absorptions per fission in core",
+        )
 
         pb.defParam(
             "axialExpansionPercent",
-            units="%",
+            units=units.PERCENT,
             description="Percent of axial growth of fuel blocks",
             default=0.0,
         )
 
-        pb.defParam("coupledIteration", units="?", description="?", default=0)
-
-        pb.defParam("fisFrac", units="?", description="?")
-
-        pb.defParam("fisRateCore", units="?", description="?")
+        pb.defParam(
+            "coupledIteration",
+            units=units.UNITLESS,
+            description="Pre-defined number of tightly coupled iterations.",
+            default=0,
+        )
 
         pb.defParam(
-            "maxdilationTotal",
-            units="?",
-            description="?",
-            default=0.0,
-            categories=["block-max"],
+            "fisFrac",
+            units=units.UNITLESS,
+            description="Percent of fissions in fertile nuclides",
         )
 
         pb.defParam(
-            "maxresidence",
-            units="?",
-            description="?",
-            default=0.0,
-            categories=["block-max"],
+            "fisRateCore",
+            units=units.UNITLESS,
```

Comment:
I am assuming that this parameter is an actual rate, not a ratio of rates, but that is just based on my interpretation of the name.

Reply:
Okay, this is a DIF3D variable. And you're right: "fissions/second".

Reply:
It is not a rate, it's the ratio of peak to average. The name is terrible, so maybe we should change that.

Below is an example DIF3D output that it would be reading to get this parameter.

```
 PEAK/AV. FISSION RATE  IN CORE    1.53080E+00
```

It's pulled by finding the phrase "FISSION RATE IN CORE" in the output, which is probably where the parameter name `fisRateCore` comes from. But it's very misleading.

I can't think of a great name for it. `peakToAvgFisRate`?

Reply:
We'll wait to change the name after it gets moved out of ARMI. Simpler than doing it now.